db/migrate/20171014204201_add_missing_indexes.rb

```diff
@@ -0,0 +1,27 @@
+class AddMissingIndexes < ActiveRecord::Migration[5.1]
+  def change
+    add_index :alerts, :subject_id
+    add_index :articles, :wiki_id
+    add_index :articles, :title
+    add_index :assignment_suggestions, :user_id
+    add_index :assignments, :article_id
+    add_index :assignments, :course_id
+    add_index :assignments, :user_id
+    add_index :assignments, :wiki_id
+    add_index :blocks, :gradeable_id
+    add_index :campaigns_courses, :campaign_id
+    add_index :campaigns_courses, :course_id
+    add_index :courses, :chatroom_id
```

Comment:
This is not a foreign key; there is no Chatroom model. The chatroom id is an id for a separate service.